pkgs/applications/blockchains/ethabi/add-Cargo-lock.patch

```diff
@@ -0,0 +1,683 @@
+diff --git a/Cargo.lock b/Cargo.lock
+new file mode 100644
+index 0000000..05e3f0f
+--- /dev/null
++++ b/Cargo.lock
+@@ -0,0 +1,677 @@
++# This file is automatically @generated by Cargo.
++# It is not intended for manual editing.
++[[package]]
```

Comment:
not critical for this PR, but it's considered best practice to version control the lock file for applications https://doc.rust-lang.org/cargo/guide/cargo-toml-vs-cargo-lock.html. Do you mind creating an issue upstream?

Reply:
They explicitly removed it at some point (link in PR desc), so I assume they have their reasons, but I can try!